packages/calcite-design-tokens/support/token-transformer/styleDictionary/transformer/value/valueColorCss.ts

```diff
@@ -14,6 +14,7 @@ function deeplyNestedColorValue(val) {
 
     if (val.includes("rgba") || color.getAlpha() === 0) {
       const hexMatch = val.match(/#[\w\d]{3,6}/g);
+      // eslint-disable-next-line no-useless-escape
```

Comment:
Can you add an explainer to this disabled line? Applies to other disabled rules.